src/banking_stage.rs

```diff
@@ -108,7 +109,9 @@ impl BankingStage {
             debug!("process_transactions");
             let results = bank.process_transactions(transactions);
             let transactions = results.into_iter().filter_map(|x| x.ok()).collect();
-            signal_sender.send(Signal::Transactions(transactions))?;
+            if let Err(_) = signal_sender.send(Signal::Transactions(transactions)) {
```

Comment:
let's not do this, let's instead first pull the Transaction->Entry code back to banking_stage, then rebase this PR on that logic